python_ta/reporters/templates/stylesheet.css

```diff
@@ -320,20 +416,21 @@ button.collapsed {
 .empty-placeholder {
   font-style: italic;
   padding-left: 20px;
+  color: var(--text-muted);
 }
 
 /* Footer styling */
 
 footer {
-  background: #2d3441;
-  color: #ffffff;
+  background: var(--text-secondary);
```

Comment:
This doesn't look very good on dark mode (contrast is too low). You could decouple this from `--text-secondary` with a separate variable.